### PR TITLE
tensorman: init at 0.1.0

### PR DESCRIPTION
**Pull request by thefenriswolf**
###### Motivation for this change
Easy management of Tensorflow containers with Docker as the backend.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers
@Lassulus
@mmahut 
opening a new PR was just easier than rebasing. I hope you don't mind.
cc @

### CODE REVIEW

**Comment by mmahut** [comment]
@GrahamcOfBorg build tensorman

File: pkgs/tools/misc/tensorman/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+ src = fetchFromGitHub {
+   owner = "pop-os";
+   repo = "tensorman";
+   rev = version;
+   sha256 = "0ywb53snvymmwh10hm6whckz7dwmpqa4rxiggd24y178jdfrm2ns";
+ };
+
+ buildInputs = [ pkgconfig openssl ];
+ cargoSha256 = "1gh5w6zzrvjk60bqaf355fagijy723rvmqjh4laksd96pmzdfwn9";
```
Comment (Lassulus) [inline pkgs/tools/misc/tensorman/default.nix]: I think there is still a space of indentation missing

Comment (thefenriswolf) [reply pkgs/tools/misc/tensorman/default.nix]: fixed

**Comment by mmahut** [comment]
@GrahamcOfBorg build tensorman

**Comment by Lassulus** [comment]
can you squash the last 2 commits and make the commit message say: `tensorman: init at 0.1.0`

**Comment by Lassulus** [comment]
the commit message needs to change. right now it says: `add tensorman` but it should say: `tensorman: init at 0.1.0`

**Comment by thefenriswolf** [comment]
fixed

**Review by Lassulus — APPROVED** [review]
tested with nix-review, seems to work

**Pull request merged**